### DISCUSSION TITLE
easy swap node option

### DISCUSSION THREAD

**Discussion by tampadesignr** [Comfy-Org/ComfyUI_frontend#1787] [Ideas]
easy swap node option

example

comfyui updates and breaks something
you find the same node that is now red on workflow in nodes panel
you drop the good node next to the red node
click on red node with an option to swap with another node on stage
choose the one you dropped next to the red
and select it
quick on to the next thing

**Comment by LukeG89** [comment]
As an alternative, you can install [comfyui-connection-helper](https://github.com/ltdrdata/comfyui-connection-helper) to easily duplicate all connections of the closest node:


https://github.com/user-attachments/assets/7b7970d2-9c2e-4c8e-a3b1-c38dea72f1a4
